deepcell/model_zoo/fpn.py

```diff
@@ -173,6 +187,11 @@ def __create_pyramid_features(backbone_dict,
     if ndim not in acceptable_ndims:
         raise ValueError('Only 2 and 3 dimensional networks are supported')
 
+    acceptable_interpolation = {'bilinear', 'nearest'}
+    if interpolation not in acceptable_interpolation:
+        raise ValueError('Interpolation mode not supported. Choose from '
+            '[\'bilinear\', \'nearest\']')
+
```

Comment:
The strings have to align to pass PEP8, also I suggest using double-quotes inside the string for better readability.

```
        raise ValueError('Interpolation mode not supported. Choose from '
                         '["bilinear", "nearest"]')
```